### PR TITLE
Scan dev monitor the queue and acquire logs to be scanned

### PR DESCRIPTION
**Pull request by Jullianos**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -49,6 +49,10 @@ gem 'sentry-ruby'
 gem 'sentry-rails'
 gem 'sentry-sidekiq'
 
+gem 'faraday'
+gem 'faraday_middleware'
+gem 'virtus'
```
Comment (vitalied) [inline Gemfile:54]: @Jullianos @AndriiMysko `virtus` is discontinued. Use `dry-types`, `dry-struct` and `dry-schema`

File: Gemfile [hunk 2]
```diff
@@ -49,6 +49,10 @@ gem 'sentry-ruby'
 gem 'sentry-rails'
 gem 'sentry-sidekiq'
 
+gem 'faraday'
+gem 'faraday_middleware'
```
Comment (vitalied) [inline Gemfile:53]: @Jullianos @AndriiMysko `faraday_middleware` is deprecated.
Why you are not using `rest-client` instead of `faraday`?

**Review by vitalied — CHANGES_REQUESTED** [review]
@Jullianos @AndriiMysko pls rebase

File: lib/travis/remote_log.rb [hunk 3]
```diff
@@ -0,0 +1,350 @@
...
+      end
+
+      message
+    end
+
+    def remote
+      @remote ||= Travis::RemoteLog::Remote.new(platform: platform)
+    end
+
+    class Client
```
Comment (vitalied) [inline lib/travis/remote_log.rb:140]: @Jullianos @AndriiMysko why are you getting logs via API HTTP calls when you have direct access to the logs DB?

**Comment by Jullianos** [comment]
@vitalied  This pull-request can be closed as it was replaced by (https://github.com/travis-ci/travis-scanner/pull/7/). Sorry for the inconvenience.

**Pull request closed without merging**